www/js/diary/list.js

```diff
@@ -155,7 +155,9 @@ angular.module('emission.main.diary.list',['ui-leaflet', 'nvd3ChartDirectives',
     */
 
     $scope.refresh = function() {
+      if ($ionicScrollDelegate.getScrollPosition().top < 5) {
```

Comment:
Can you add an issue to investigate this further? It seems really wrong that it is required.